Add pytest-responses to enable responses automatically refs #49

Adrian-Tamas opened the pull request.

La0 commented on bot/requirements-dev.txt:
```diff
@@ -1,3 +1,4 @@
 pre-commit==1.18.2
 pytest==5.1.1
 responses==0.10.6
+pytest-responses==0.4.0
```
We try to keep this file ordered alphabetically: please move that line before `responses`

Adrian-Tamas replied:
fixed

Adrian-Tamas commented:
@La0 I fixed the order and also removed all the other instances of @responses.activate . Sorry about missing thouse. I did realize that I should have checked in more places but for some reason it was only late yesterday. I have fixed that and squashed the commits into 1.
